docs/wallets/decrediton/upgrading-decrediton.md

```diff
@@ -0,0 +1,16 @@
+# Upgrading Decrediton
+
+To upgrade Decrediton, simply download the new version and install it. You do not need to delete your existing version or perform any additional steps. When you open the upgraded version, your existing wallets and accounts will appear. 
+
+## Upgrading steps
+
+1. Download latest version. The latest downloads can be found at [https://decred.org/downloads/](https://decred.org/downloads/)
+or by clicking on the `Update Available` button in the upper right-hand corner of Decrediton.
+    <br>
+    ![Decrediton update link](/img/decrediton/upgrading/update-available.png)
+    Click on the link for your OS and download the installer.
+    ![Decrediton update link](/img/decrediton/upgrading/download-link.png)
+2. Close your current version of Decrediton (if open) and click on the downloaded installer. Follow the installation instructions as you would for any other program. 
```

Comment:
Remember to only use `1.` for every item of a numbered list, and then markdown will increment the numbers automatically.

Reply:
Right. Good best practice. Done.